tests/contrib/models/test_tiny_vit.py

```diff
@@ -35,6 +37,7 @@ def test_gradcheck(self): ...
     @pytest.mark.skip("not implemented")
     def test_module(self): ...
 
+    @pytest.mark.skipif(sys.version_info.major == 3 and sys.version_info.minor == 8, reason="not working for py3.8")
```

Comment:
Similar to torch_version utils, we could have python_version

Reply:
i think we can use [the version_info directly](https://docs.python.org/3/library/sys.html#sys.version_info), this one we can rewrite like
```suggestion
    @pytest.mark.skipif(sys.version_info[:2] == (3, 8), reason="not working for py3.8")
```